### ISSUE TITLE
`pm2 id` doesn't seem to work in pm2 v2

### ISSUE THREAD

**Issue by pepkin88** [Unitech/pm2#2394]
#### Expected behaviour

`pm2 id name` returns a JSON encoded array of app ids, just like it was in version 1.x.

#### Actual behaviour

Nothing is returned (0 bytes). I tried with cluster mode and fork mode, same result.

#### Steps to reproduce

Start an app with `pm2 start app.js` (assuming `app.js` exists and contains a code to run a server).
Run `pm2 ls`, it should list the newly created app as `app`.
Run `pm2 is app`. In v2 it returns nothing.

#### Software versions used

```
OS         : Ubuntu 14.04
node.js    : v5.9.0
PM2        : v2.0.12
```

#### PM2 Log output

```
2016-09-20 15:53:53: Starting execution sequence in -fork mode- for app name:Everytap API id:1
2016-09-20 15:53:53: App name:Everytap API id:1 online
2016-09-20 17:36:10: Starting execution sequence in -cluster mode- for app name:Everytap id:0
2016-09-20 17:36:10: App name:Everytap id:0 online
2016-09-20 17:36:21: Stopping app:Everytap id:_old_0
2016-09-20 17:37:11: Starting execution sequence in -cluster mode- for app name:Everytap id:0
2016-09-20 17:37:12: App name:Everytap id:0 online
2016-09-20 17:37:23: Stopping app:Everytap id:_old_0
2016-09-20 17:47:08: Starting execution sequence in -cluster mode- for app name:Everytap id:0
2016-09-20 17:47:08: App name:Everytap id:0 online
2016-09-20 17:47:19: Stopping app:Everytap id:_old_0
2016-09-20 17:47:20: Process with pid 6150 still not killed, retrying...
2016-09-20 17:47:20: Process with pid 6150 still not killed, retrying...
2016-09-20 17:47:20: Process with pid 6150 still not killed, retrying...
2016-09-20 17:47:21: Process with pid 6150 still not killed, retrying...
2016-09-20 17:47:21: Process with pid 6150 still not killed, retrying...
2016-09-20 17:47:21: Process with pid 6150 still not killed, retrying...
2016-09-20 17:47:21: Process with pid 6150 still not killed, retrying...
2016-09-20 17:55:41: Stopping app:Everytap API id:1
2016-09-20 17:55:41: Stopping app:Everytap API id:2
2016-09-20 17:55:41: Stopping app:Everytap API id:3
2016-09-20 17:55:41: Stopping app:Everytap API id:4
2016-09-20 17:55:41: App [Everytap API] with id [3] and pid [29663], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: App [Everytap API] with id [4] and pid [29718], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: App [Everytap API] with id [1] and pid [14498], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: App [Everytap API] with id [2] and pid [29658], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:1
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:2
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:3
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:4
2016-09-20 17:55:41: App name:Everytap API id:1 online
2016-09-20 17:55:41: App name:Everytap API id:2 online
2016-09-20 17:55:41: App name:Everytap API id:3 online
2016-09-20 17:55:41: App name:Everytap API id:4 online
2016-09-20 17:55:41: Stopping app:Everytap API id:5
2016-09-20 17:55:41: Stopping app:Everytap API id:6
2016-09-20 17:55:41: Stopping app:Everytap API id:7
2016-09-20 17:55:41: Stopping app:Everytap API id:8
2016-09-20 17:55:41: App [Everytap API] with id [5] and pid [29723], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: App [Everytap API] with id [7] and pid [29783], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: App [Everytap API] with id [8] and pid [29788], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: App [Everytap API] with id [6] and pid [29729], exited with code [0] via signal [SIGINT]
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:5
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:6
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:7
2016-09-20 17:55:41: Starting execution sequence in -fork mode- for app name:Everytap API id:8
2016-09-20 17:55:41: App name:Everytap API id:5 online
2016-09-20 17:55:41: App name:Everytap API id:6 online
2016-09-20 17:55:41: App name:Everytap API id:7 online
2016-09-20 17:55:41: App name:Everytap API id:8 online
```

**Comment by Unitech** [comment]
fixed

```
$ npm install Unitech/pm2#development -g
```

**Comment by Unitech** [comment]
landed in latest public release (2.0.15)

```bash
$ npm install pm2 -g
```

**Comment by AjaxOdessa** [comment]
I'm experiencing the same issue with
Ubuntu 14.04.4 LTS
NodeJS 6.6.0
pm2 2.0.18
```
ubuntu@srv:/var/www/srv$ pm2 id socket
[ 0 ]
ubuntu@srv:/var/www/srv$ pm2 show socket
 Describing process with id 0 - name socket 
┌───────────────────┬───────────────────────────────────────────────────┐
│ status            │ online                                            │
│ name              │ socket                                            │
│ restarts          │ 4                                                 │
│ uptime            │ 16s                                               │
│ script path       │ /var/www/srv/socket.js                            │
│ script args       │ N/A                                               │
│ error log path    │ /var/www/srv/storage/logs/socket_js.err.log       │
│ out log path      │ /var/www/srv/storage/logs/socket_js.out.log       │
│ pid path          │ /home/ubuntu/.pm2/pids/socket.pid                 │
│ interpreter       │ node                                              │
│ interpreter args  │ N/A                                               │
│ script id         │ 0                                                 │
│ exec cwd          │ /var/www/srv                                      │
│ exec mode         │ cluster_mode                                      │
│ node.js version   │ 6.6.0                                             │
│ watch & reload    │ ✔                                                 │
│ unstable restarts │ 0                                                 │
│ created at        │ 2016-10-18T15:44:45.717Z                          │
└───────────────────┴───────────────────────────────────────────────────┘
 Code metrics value 
┌────────────┬────────┐
│ Loop delay │ 0.45ms │
└────────────┴────────┘
 Add your own code metrics: http://bit.ly/code-metrics
 Use `pm2 logs socket [--lines 1000]` to display logs
 Use `pm2 monit` to monitor CPU and Memory usage socket
ubuntu@vptaxdev:/var/www/srv$ pm2 -v
2.0.18
ubuntu@srv:/var/www/srv$
```
